graphql_query_derive/src/lib.rs

```diff
@@ -75,8 +95,14 @@ fn impl_gql_query(input: &syn::DeriveInput) -> Result<TokenStream, failure::Erro
     let cargo_manifest_dir =
         ::std::env::var("CARGO_MANIFEST_DIR").expect("CARGO_MANIFEST_DIR env variable is defined");
 
-    let query_path = extract_attr(input, "query_path")?;
-    let schema_path = extract_attr(input, "schema_path")?;
+    let module_name = Ident::new(&input.ident.to_string().to_snake_case(), Span::call_site());
+    let struct_name = &input.ident;
+
+    let query_path = extract_attr(input, "query_path")
+        .or_else(|_err| implicitly_discover_query_name(&struct_name.to_string()))?;
```

Comment:
There's an error message line 60 in this file that gets printed if the file isn't found, we should update it with info on how we try to find the file explicitly, otherwise it will be confusing. The problem is that in `read_file` we don't know whether the path was specified implicitly or just inferred so we can't adapt the error message. Do you have an idea? I'll try to think of something too.